src/store/userStore.js

```diff
@@ -43,6 +46,18 @@ export const useUserStore = defineStore("user", {
                 uuid     : this.userObj.uuid
             });
         },
+        async createdRelation() {
+            const response = await createRelation({
+                contentUuid : this.uuidContent,
+                op          : "Content",
+                relation    : "COMPLETED"
```

Comment:
if they have a failing score would should they not have a rel made as unCompleted with the score they currently have